in_text/text_data_release.yml

```
@@ -435,16 +441,16 @@ entities:
     -
       attr-label: Func_Perf
       attr-def: >-
-        XX
+        the calculated fucntional performance
       attr-defs: >-
         This release
       data-min: -0.227543205
       data-max: 0.145496753
-      data-units: XX
+      data-units: unitless since transfer entropy is of nats then normalized by the entropy of the sink variable
     -
       attr-label: model
       attr-def: >-
-        XX
+        the name of the model that is being assesseed
```

does this correspong with run_id?
```suggestion
        The name of the model that is being assessed.
```